diff --git a/src/backend/cpu/kernel/sort.hpp b/src/backend/cpu/kernel/sort.hpp
index dd842dce..4f2a8853 100644
--- a/src/backend/cpu/kernel/sort.hpp
+++ b/src/backend/cpu/kernel/sort.hpp
@@ -27,7 +27,7 @@ void sort0Iterative(Param<T> val, bool isAscending)
     // initialize original index locations
     T *val_ptr = val.get();
 
-    function<bool(T, T)> op = std::greater<T>();
+	std::function<bool(T, T)> op = std::greater<T>();
     if(isAscending) { op = std::less<T>(); }
 
     T *comp_ptr = nullptr;
